resources/js/services/authService.js

```diff
@@ -51,3 +51,12 @@ export const logout = async (token) => {
       store.dispatch(logoutAction());
     });
 };
+
+export const checkValidation = async (token,fullname, number, date, email, phone) => {
+  const response = await getAxiosInstance(token)
+    .post('checkValidation',{fullname, number, date, email, phone});
+
+ 
+  console.log(response);
```

Comment:
@hariidev Better to remove console logs when releasing code.

Reply:
just added to check the response. will remove it